### DISCUSSION TITLE
Problem with the select label-color and options-selected-class: a possible bug.

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#13970] [General - Components / Directives / etc]
I want to selected columns be in the primary color and the non-visible columns in grey. But it shows always in white in dark mode.

<q-select
          v-model="visibleColumns"
          multiple
          outlined
          dense
          options-dense
          :display-value="$q.lang.table.columns"
          emit-value
          map-options
          :options="columns"
          option-value="name"
          options-cover
          style="min-width: 150px"
          color="grey-5"
          options-selected-class="text-primary"
        />

**Comment by MilosPaunovic** [comment]
Please provide a Codepen reproduction.

**Comment by meo1** [comment]
https://codepen.io/meo1meo1/pen/wvmgMbN?editors=101

I want the list of invisible column names in select box of the table to be grey out. I tried several methods but I cannot succeed.

`color="grey-5" 
  options-selected-class="primary"`

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/JjLEbRr?editors=1011](https://codepen.io/milospaunovic/pen/JjLEbRr?editors=1011)

```html
content-class="text-grey-5"
options-selected-class="text-secondary"
```

**Reply by meo1** [reply]
It works in the light mode, but does not work in the dark mode. Even I set the color of the popup-content-class to another value, it shows white text.

**Comment by meo1** [comment]
Your solution works only in the light mode. Here is the code for dark mode:

https://codepen.io/meo1meo1/pen/yLKgrpw?editors=1011
